TLM/TLM/TrafficManagerMod.cs

```diff
@@ -75,6 +90,9 @@ public class TrafficManagerMod : IUserMod {
                     Log.InfoFormat("Mono version: {0}", displayName.Invoke(null, null));
                 }
             }
+
+            Instance = this;
```

Comment:
Where is this getting used exaclty? When I look at the code in VS2019, it's not showing any references to it.

I can see down on line 110 (in updated `TrafficManagerMod.cs`) that it gets set to `null` in the `OnDisabled()` method, but other than that it's not getting used.

Was it supposed to be referenced in line 105 in the `OnDisabled()` method?

Reply:
see
 TLM/TLM/State/SerializableDataExtension.cs :29
VersionLabel.cs :14